lib/runner/folder-walk.js

```diff
@@ -176,6 +177,11 @@ class Walker {
           fullPath = [fullPath];
         }
 
+        // cucumber-runner manages it's own set of filters
+        if (this.usingCucumber) {
+          return resolve(fullPath);
```

Comment:
Is this necessary? Do we pass in the nightwatch filter setting to Cucumber? How do we pass in filter string to cucumber?

Reply:
Yes, we don't want to filter the process at Nightwatch's end as that would filter on step definitions (cucumber  has its own CLI params like `tags` to filter out feature files)

Reply:
Okay